### ISSUE TITLE
Clarification on 'id' in annotation dict

### ISSUE THREAD

**Issue by kauravin** [Duankaiwen/CenterNet#115]
Hi @Duankaiwen, I've been able to run several successful experiments using the code on the COCO dataset. However, for a custom dataset, I'm a bit unclear about the box_ids. Each individual dict in the annotations object (in instances_<>.json) corresponds to a single instance (out of several possible instances for a given class, for a given image) and is of this format:
bb_dict = {
'segmentation': [], 
'area': area of the bounding box
'iscrowd': 0,
'image_id': image_id,
'bbox': bb_coordinates,
'category_id': category_id,
'id': annotation_id
}
If for a given annotation mask in an image, there are 10 bounding boxes, then I'm assuming (from COCO's structure) there would be 10 such dictionaries. However, all these 10 dictionaries share the same 'id' (annotation ID) which is unique for the given class. Is my understanding correct? I also occasionally see 0 push, pull losses on my own dataset. Please let me know what you think. 

*In short, is the 'id' in the above dict unique per instance, or unique per class for a single image? Thanks!*
